eos/b-decays/b-to-k-nu-nu_TEST.cc

```diff
@@ -0,0 +1,79 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2021 Danny van Dyk
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#include <test/test.hh>
+#include <eos/observable.hh>
+#include <eos/b-decays/b-to-psd-nu-nu.hh>
+#include <eos/utils/complex.hh>
+#include <eos/utils/wilson-polynomial.hh>
+
+#include <array>
+#include <cmath>
+#include <iostream>
+#include <fstream>
+#include <limits>
+#include <string>
+#include <vector>
+
+
+using namespace test;
+using namespace eos;
+
+class BToKDineutrinoTest :
+    public TestCase
+{
+    public:
+        BToKDineutrinoTest() :
+            TestCase("b_to_k_nu_nu_test")
+        {
+        }
+
+        virtual void run() const
+        {
+            {
+                Parameters p = Parameters::Defaults();
+                p["CKM::abs(V_tb)"]           =  1.00;
+                p["CKM::abs(V_ts)"]           =  4.00e-2;
+                p["B->K::alpha^f+_0@BSZ2015"] = +3.2909e-01;
+                p["B->K::alpha^f+_1@BSZ2015"] = -8.6695e-01;
+                p["B->K::alpha^f+_2@BSZ2015"] = +6.0957e-03;
+                p["mass::B_u"]                =  5.2796;
+                p["mass::K_u"]                =  4.9368e-01;
+
+                Options oo
+                {
+                    { "model",        "CKM"     },
+                    { "form-factors", "BSZ2015" },
+                    { "D",            "s"       },
+                    { "q",            "u"       },
+                    { "I",            "1/2"     }
+                };
+
+                BToPseudoscalarDineutrino d(p, oo);
+
+                const double eps = 1e-8;
+
+                TEST_CHECK_NEARLY_EQUAL(2.05845e-06, d.integrated_branching_ratio( 0.00,  8.00), eps);
```

Comment:
I find slightly inconsistent to set `eps = 1e-8` and then give number with only 5 decimals, no?

Reply:
Forget about this, it's not a relative error. I would still prefer to use relative errors

Reply:
We can go for a relative error, no problem as long as the values are non-zero.

Reply:
Done. `1e-4` for all as relative error.